### PR TITLE
Add support for Metamako MOS and Exablaze Fusion devices. IPv4-only fixes

### PR DESCRIPTION
**Pull request by gs-kamnas**
This PR adds support for discovering and polling Metamako devices running Metamako's MOS operating system, as well as Exablaze Fusion devices. Additionally, this changeset resolves an issue where validation fails on systems where IPv6 is disabled by unloading/omitting the relevant kernel modules (as opposed to disabling via sysctl).

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: includes/definitions/linux.yaml [hunk 1]
```diff
@@ -34,3 +34,4 @@ register_mibs:
 discovery:
     - sysObjectID: .1.3.6.1.4.1.8072.3.2.10
     - sysDescr_regex: '/^Linux/'
+    - sysObjectID_except: .1.3.6.1.4.1.43296.
```
Comment (laf) [inline includes/definitions/linux.yaml]: You shouldn't need this here.

As your device is using `.1.3.6.1.4.1.43296.` for sysObjectID then it wouldn't match anyway.

File: includes/definitions/discovery/exablaze-fusion.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+mib: EXALINK-FUSION-MIB
+modules:
+    sensors:
+        pre-cache:
```
Comment (laf) [inline includes/definitions/discovery/exablaze-fusion.yaml]: You can drop the `pre-cache` stuff here. It's only needed if you need access to other snmp data.

File: includes/polling/os/exablaze-fusion.inc.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+#EXALINK-FUSION-MIB::fusionInfoSerial = STRING: EXAFSN-B-00402
```
Comment (laf) [inline includes/polling/os/exablaze-fusion.inc.php]: You can drop the comments :)

File: includes/polling/os/exablaze-fusion.inc.php [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+#EXALINK-FUSION-MIB::fusionInfoVersion = STRING: 1.5.0
+#EXALINK-FUSION-MIB::fusionInfoBoard = STRING: EBFSN-03
+#EXALINK-FUSION-MIB::fusionInfoSoftware = STRING: 2016-09-05 18:16:58 +1000 (c4573c4)
+
+$hardware = snmp_get($device, "fusionInfoBoard", "-Ovq", "EXALINK-FUSION-MIB");
```
Comment (laf) [inline includes/polling/os/exablaze-fusion.inc.php]: Please convert these four `snmp_get()` calls to a single `snmp_get_multi_oid()` call.

File: includes/definitions/linux.yaml (hunk 1 above)
Comment (gs-kamnas) [reply includes/definitions/linux.yaml]: I was having issues here because Metamako devices have a sysDescr beginning with "Linux" though.

**Comment by gs-kamnas** [comment]
@laf Will refactor and split into 2 PRs as requested.

**Pull request closed without merging**